### ISSUE TITLE
History search string field

### ISSUE THREAD

**Issue by abiacco** [Tautulli/Tautulli-Remote#96]
**Is your feature request related to a problem? Please describe.**
The 2.0 version of the app does not have a search string field in the History menu like the 1.x app did

**Describe the solution you'd like**
The search string field to be added to the History menu in the 2.0 version of the app

**Describe alternatives you've considered**
Waiting

**Additional context**

**Comment by Dulanic** [comment]
I'm with you. That's my biggest use of tautulli is when my wife says I think we watched this recently and we argue I want Tautulli to back me up! :)

**Comment by TheMeanCanEHdian** [comment]
I have created initial functionality for this here c61b7984992f311a1911ad789b6bb85d175fb507, and it will be available starting in V3 Alpha 12.

If you are on Android feel free to [install Alpha 12](https://github.com/Tautulli/Tautulli-Remote/issues/168) once it is available to test this. Otherwise, it will be available on both platforms at a later date.

I will close this issue once Alpha 12 has been released.
